tests/behat/bootstrap/FeatureContext.php

```
@@ -22,13 +27,110 @@
+          ->save();
+      }
+    }
+  }
+
+  /**
+   * Gets the config factory service.
+   *
+   * @return \Drupal\Core\Config\ConfigFactoryInterface
+   *   The config factory service.
+   *
+   * @SuppressWarnings(PHPMD.StaticAccess)
```

PHPStan (2.0.3)

PHPDoc tag @SuppressWarnings has invalid value ((PHPMD.StaticAccess)): Unexpected token ".StaticAccess)", expected ')' at offset 171 on line 7


(phpDoc.parseError)